inst/table.m

```diff
@@ -424,19 +424,26 @@ function prettyprint (this)
     ## -*- texinfo -*-
     ## @node table.size
     ## @deftypefn {Method} {@var{sz} =} size (@var{obj})
+    ## @deftypefnx {Method} {[@var{nr}, @var{nv}] =} size (@var{obj})
     ##
     ## Gets the size of a table.
     ##
     ## For tables, the size is [number-of-rows x number-of-variables].
     ## This is the same as @code{[height(obj), width(obj)]}.
     ##
     ## @end deftypefn
-    function out = size (this, dim)
-      if nargin == 1
-        out = [height(this), width(this)];
+    function [out, nv] = size (this, dim)
+      if nargout == 1
+        if nargin == 1
+          out = [height(this), width(this)];
+        elseif dim == 2
+          out = width(this);
+        else
+          out = height(this);
+        end
       else
-        sz = size (this);
-        out = sz(dim);
+        out = height(this);
+        nv  = width(this);
```

Comment:
Got an extra space here. Tablicious and the standard GNU Octave style generally do not do equal-sign alignment.

Reply:
Cool beans! No rush - it's the weekend, and this is a volunteer project. I'll wait on your PR revision before making any other moves here.